src/ru/skypro/Main.java

```diff
@@ -3,6 +3,74 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        int clientOS = 0;
+        if (clientOS == 0) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        } else {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+
+        int clientOSNew = 0;
+        int productionYear = 2014;
+
+        if (clientOSNew == 0 && productionYear >= 2015) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        }
+        if (clientOSNew == 0 && productionYear < 2015) {
+            System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+        }
+        if (clientOSNew == 1 && productionYear >= 2015) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+        if (clientOSNew == 1 && productionYear < 2015) {
+            System.out.println("Установите облегченную версию приложения для Android по ссылке");
+        }
+
+
+        int year = 2021;
+        if (((year % 4 == 0) && (year % 100 != 0)) || (year % 400 == 0)) {
+            System.out.println("Год " + year + " является високосным.");
+        } else {
+            System.out.println("Год " + year + " является невисокосным.");
+        }
+
+
+        int deliveryDistance = 95;
+        if (deliveryDistance < 20) {
+            System.out.println("Доставка займет 1 день");
+        } else if (deliveryDistance >= 20 && deliveryDistance < 60) {
```

Comment:
поскольку мы имеем единый оператор, то данную и ниже ветвь можно упростить, так как первое условие автоматом выполняется из-за ложности условия выше
```suggestion
        } else if (deliveryDistance < 60) {
```